Documents/Changelog/Changelog.md

```diff
@@ -3,6 +3,14 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Implemented [#1507](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1507), **[Breaking Change]** `KryptonThemeComboBox`, `KryptonThemeListBox` & `KryptonRibbonGroupThemeComboBox`:
```

Comment:
Breaking change description goes into a different section

Reply:
> Breaking change description goes into a different section

What about the others in the list. That's how I came to putting it in with the rest.

Some help please as I don't know where that would be.